simulation.config

```diff
@@ -1,10 +1,14 @@
-nodeCount = 10
-iterations = 10
+nodeCount = 5
+iterations = 5
 pace = 1
 alpha = 12
-txmin = 3
-signaturesThreshold = 4
-initialBalance = 20
+txmin = 2
+signaturesThreshold = 2
+initialBalance = 30
 levels = 30
 Mode = True
 validationFees = 1
+token = 19
+#if greater than equal to 10 then passing the current contract
```

Comment:
`#if greater than equal to 10 then passing the current contract` this needs more clarification, also the term `current` contract is not very clear.